demos/fleet_provisioning/fleet_provisioning_with_csr/demo_template.json

```diff
@@ -0,0 +1,52 @@
+ {
+   "Parameters": {
+     "SerialNumber": {
+       "Type": "String"
+     },
+     "AWS::IoT::Certificate::Id": {
+       "Type": "String"
+     }
+   },
+   "Resources": {
+     "certificate": {
+       "Properties": {
+         "CertificateId": {
+           "Ref": "AWS::IoT::Certificate::Id"
+         },
+         "Status": "Active"
+       },
+       "Type": "AWS::IoT::Certificate"
+     },
+     "policy": {
+       "Properties": {
+         "PolicyName": "demo_thing_policy"
+       },
+       "Type": "AWS::IoT::Policy"
+     },
+     "thing": {
+       "OverrideSettings": {
+         "AttributePayload": "MERGE",
+         "ThingGroups": "DO_NOTHING",
+         "ThingTypeName": "REPLACE"
+       },
+       "Properties": {
+         "AttributePayload": {},
+         "ThingGroups": [],
+         "ThingName": {
+           "Fn::Join": [
+             "",
+             [
+               "fp_demo_",
+               {
+                 "Ref": "SerialNumber"
+               }
+             ]
+           ]
+         },
+         "ThingTypeName": "fp_demo_things"
+       },
+       "Type": "AWS::IoT::Thing"
+     }
+   },
+   "DeviceConfiguration": {}
```

Comment:
You may want to show some information of device configuration to send down to the device. You can use the example provided by the example docs: https://docs.aws.amazon.com/iot/latest/developerguide/provision-template.html#fleet-provision-template